### ISSUE TITLE
Get the User's Currently Playing Track in Private Session

### ISSUE THREAD

**Issue by saeedesmaili** [spotify/web-api#1375]
How can we retrieve a user's currently playing track, while he/she is using the private session?

Assume that the user is willing to give such access.

**Comment by jscholes** [comment]
> Assume that the user is willing to give such access.

I mean, the user isn't willing to give such access, hence why their session is private.  That's the whole point of a private session, so why would you expect it to be violated for your app?

**Comment by saeedesmaili** [comment]
@jscholes Yes, but in my case, the user has some followers (and he can't block them), so he doesn't want them to see the currently playing trach and uses the private session because of that. But he wants to share the currently playing track with some other close friends in a third party app.

**Comment by jscholes** [comment]
This sounds like a feature for Spotify to implement within their UI, in terms of having fine-grain control over whom data is shared with and a follower grouping system.  But the use case also sounds really niche, so I doubt it will ever happen.
